evap/staff/templates/staff_semester_grade_reminder.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'staff_semester_base.html' %}
+
+{% load evaluation_filters %}
+
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="breadcrumb-item">{% trans 'Grade Publish Reminder' %}</li>
+{% endblock %}
+
+{% block content %}
+    {{ block.super }}
+
+    <h3>
+        {{semester.name}}
+    </h3>
+
+    <div class="card">
+        <div class="card-body">
+            {% for responsible, courses, delegates in responsible_list %}
+                <p>
+                    <b>{{responsible.full_name}}</b>
```

Comment:
please add spaces after and before the braces here and for all other variable uses like this:
`{{ responsible.full_name }}`